descriptors/ModuleDescriptor-template.json

```diff
@@ -326,7 +336,8 @@
         "consortia.sharing-instances.collection.get",
         "consortia.publications.item.post",
         "consortia.publications.item.get",
-        "consortia.publications-results.item.get"
+        "consortia.publications-results.item.get",
+        "consortia.publications-results.item.delete"
```

Comment:
should be `consortia.publications.item.delete`

Reply:
Fixed.